addons/website_payment/controllers/portal.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
```

Comment:
always try to use safeversion...  `from odoo.tools.json import scriptsafe as json_safe`

Already done this comment :/